app/plugins/prometheus.md

```diff
@@ -0,0 +1,171 @@
+---
+id: page-plugin
+title: Plugins - Prometheus
+header_title: Prometheus
+header_icon: https://konghq.com/wp-content/uploads/2018/06/prometheus.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Usage
+    items:
+      - label: Log Format
+description: |
+  This plugin exposes metrics in [Prometheus Exposition format](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md). It allows you to integrate with your existing monitoring and alerting system.
```

Comment:
Can we change this to something like:
 ```
Expose metrics related to Kong and the upstream services in 
[Prometheus](https://prometheus.io/docs/introduction/overview/) Exposition format 
which can be scrapped by a Prometheus Server.
```

This is to keep the description aligned with how other plugins do it and the link to Prometheus website is more reliable than the Github link.

Reply:
Done